listings/src/test/java/com/okta/developer/listings/AirbnbListingMvcTest.java

```diff
@@ -59,20 +56,5 @@ public void save_withValidJwtToken_returnsCreated() throws Exception {
                 .andDo(print())
                 .andExpect(status().isCreated())
                 .andExpect(jsonPath("$.id").isNotEmpty());
-
-    }
-
-    @Test
-    public void collectionGet_withOpaqueToken_returnsOk() throws Exception {
-        this.mockMvc.perform(get("/listing").with(opaqueToken())).andExpect(status().isOk());
     }
-
-    @Test
-    public void collectionGet_withInvalidJWtToken_returnsOk() throws Exception {
-        this.mockMvc.perform(get("/listing").with(jwt()
-        .jwt(jwt -> jwt.claim("exp", Instant.MIN)
-                .claim("iss", "invalid")
-                .claim("aud", "invalid")))).andExpect(status().isOk());
-    }
-
-}
```

Comment:
@indiepopart These tests weren't part of the post, that's why I removed them. Should we add them to the blog post?